plugins/woocommerce/includes/class-wc-tracker.php

```diff
@@ -776,6 +779,33 @@ public static function get_cart_checkout_info() {
 		);
 	}
 
+	/**
+	 * Get info about the Mini Cart Block.
+	 *
+	 * @return array
+	 */
+	public static function get_mini_cart_info() {
+		$mini_cart_block_name = 'woocommerce/mini-cart';
+		$mini_cart_block_data = wc_current_theme_is_fse_theme() ? WC_Blocks_Utils::get_block_from_template_part( $mini_cart_block_name, 'header' ) :
+			array_reduce(
+				get_option( 'widget_block' ),
+				function ( $acc, $block ) use ( $mini_cart_block_name ) {
+					$parsed_blocks = ! empty( $block ) && is_array( $block ) ? parse_blocks( $block['content'] ) : array();
+					if ( ! empty( $parsed_blocks ) && $mini_cart_block_name === $parsed_blocks[0]['blockName'] ) {
+						array_push( $acc, $parsed_blocks[0] );
+						return $acc;
+					}
+					return $acc;
+				},
+				array()
+			);
```

Comment:
Do you think we should move this into `WC_Blocks_Utils` as another util function? I think it would make the code simpler.